tests/release/service/tenant_pipelines.go

```diff
@@ -21,7 +21,7 @@ import (
 	tektonv1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
 )
 
-var _ = framework.ReleaseServiceSuiteDescribe("Release service tenant pipeline", Label("release-pipelines", "tenant"), func() {
+var _ = framework.ReleaseServiceSuiteDescribe("Release service tenant pipeline", Label("release-service", "tenant"), func() {
```

Comment:
Pretty sure this is covered in another PR as well

Reply:
The same as above.